log/options.go

```diff
@@ -9,11 +11,13 @@ var defaultConfig = Config{
 	OutputJSON: false,
```

Comment:
We should add the default `TimeFormat: time.Kitchen` for keeping the same behavior.

Reply:
done